### DISCUSSION TITLE
Persistent Notification

### DISCUSSION THREAD

**Discussion by stanpanchenkosommoio** [buildnatively/community#14] [Ideas]
### Use case

```Text
Would it be possible to display a persistent notifications with custom icon, title, and message?
Should be able to turn on and off.
```

### Idea

```Text
Would it be possible to display a persistent notifications with custom icon, title, and message?
Should be able to turn on and off.
```

**Comment by vdiaza** [comment]
In my specific use case it would be something like "App is running" which cannot be cleared out.  I want to do something similar to "Waze". The idea is that can be activated and deactivated through code. Of course if the app is closed it should be dismissed.

It should be customizable (icon, title, description, link), or maybe the icon can be the same as the app. if no link, just return to the app.
![Waze](https://imgur.com/qSkvaTh.png)

**Comment by romanfurman6** [comment]
@vdiaza thanks for sharing we will double check it 🙌
